frame/contracts/src/lib.rs

```diff
@@ -98,18 +98,18 @@ pub mod weights;
 #[cfg(test)]
 mod tests;
 
-pub use crate::{
-	exec::Frame,
-	pallet::*,
-	schedule::{HostFnWeights, InstructionWeights, Limits, Schedule},
-};
 use crate::{
-	exec::{AccountIdOf, ExecError, Executable, Stack as ExecStack},
+	exec::{AccountIdOf, ExecError, Executable, Stack as ExecStack, StorageHash},
 	gas::GasMeter,
 	storage::{meter::Meter as StorageMeter, ContractInfo, DeletedContract, Storage},
 	wasm::{OwnerInfo, PrefabWasmModule},
 	weights::WeightInfo,
 };
+pub use crate::{
+	exec::{FixSizedKey, Frame, VarSizedKey},
+	pallet::*,
+	schedule::{HostFnWeights, InstructionWeights, Limits, Schedule},
+};
```

Comment:
Why change the position? Let's keep the `pub` exports on top and not in-between the other imports.